_includes/checklist.md

```diff
@@ -24,7 +24,9 @@ Everything in this section needs to be completed before the project will be sche
     - [ ] Add Rules of Engagement (RoE) template
         * Search [this page](https://insite.gsa.gov/portal/content/627238) for "Rules of Engagement (RoE) 90-Day LATO Penetration Test TEMPLATE", even if this isn't for a 90-day LATO.
     - [ ] Add [System Security Plan (SSP)](https://pages.18f.gov/before-you-ship/ato/ssp/) template
+        * For Low systems on cloud.gov, use [this template](https://docs.google.com/a/gsa.gov/document/d/1tVbH39TFfvSaBbjWfLaR3GLOuvsLuhLFJ75xKowEV5c/edit?usp=sharing)
```

Comment:
Still waiting on access to this, so let's wait to merge until I can confirm this link is right.